plenum/test/test_node.py

```diff
@@ -410,6 +410,8 @@ def restart_clientstack(self):
         time.sleep(0.2)
         logger.debug("Starting clientstack on node {}".format(self))
         self.clientstack.start()
+        # Sleep to allow disconnected clients to reconnect before sending replies from the server side.
```

Comment:
Should we use `restart` method from clientstack here?

Reply:
Of course, done.